src/classes/UTIL_HtmlOutput_CTRL.cls

```diff
@@ -54,6 +54,7 @@ public with sharing class UTIL_HtmlOutput_CTRL {
     /** @description The map of allowed urls and their temporary substitution values */
     private static final Map<String, String> SUBSTITUTION_BY_ALLOWED_URL = new Map<String, String> {
         '<a href="https://powerofus.force.com/' => '|hubURL|',
+        '<a href="/lightning/setup/' => '|lightningSetupURL|',
```

Comment:
@ptewson-sfdo - Please note, I had to add this line so that we can link to a lightning setup page.